Chessie.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
-VisualStudioVersion = 12.0.31101.0
+# Visual Studio 2013
```

Comment:
You don't need to necessarily commit certain lines. In almost every Git GUI Tool (if you aren't familiar with the Git CLI) you can select which lines you want and which lines you want to discard (or keep staged).